020-LinuxFundamentals/Student/Challenge-12.md

```diff
@@ -0,0 +1,40 @@
+# Challenge 12 - Setting up a webserver 
+
+[< Previous Challenge](./Challenge-11.md) - **[Home](../README.md)** 
+
+## Description
+
+In this challenge we will be setting up a webserver and deploy a simple PHP application into it. As a plus, you can add SSL to ensure security requirements.
+
+- Download the sample application [from here](./resources/simple-php-app.tar.gz) to your home directory
+- Extract the content of simple-php-app.tar.gz on our home directory
+- Install nginx-core
+- Install php7.4-fpm
+- Configure Nginx
+
+## Success Criteria
+
+1. Make sure you have the file simple-php-app-tar.gz within your `~`
```

Comment:
Only 3 & 4 are needed here for success criteria.  The first are implied when doing this the first time.